src/main/java/com/codeborne/selenide/Selenide.java

```diff
@@ -101,6 +101,28 @@ public static void open(String relativeOrAbsoluteUrl, String domain, String logi
     mockModalDialogs();
   }
 
+  /**
+   *
+   * The main starting point in your tests.
+   * <p>
+   * Open browser and pass authentication using build-in proxy.
+   * <p>
+   * A common authenticationType is "Basic". See Web HTTP reference for other types.
+   * <p>
+   * {@code Configuration.fileDownload == Configuration.FileDownloadMode.PROXY;} - is required configuration.
+   *
+   * @param relativeOrAbsoluteUrl
+   * @param authenticationType
+   * @param login
+   * @param password
```

Comment:
I would remove all useless lines from javadoc, like `@param login` without description.

Reply:
@asolntsev 
We already had such empty javadocs lines for old `open(relativeOrAbsoluteUrl, domain, login, password)` method. 
I do it in one style during new implementation.
In this case we had to reformat all methods related to this point.

Reply:
I agree, we can cleanup javadoc later.